pages/dashboard/history.tsx

```diff
@@ -0,0 +1,12 @@
+import DashboardLayout from 'sections/dashboard/DashboardLayout';
+import History from 'sections/dashboard/History';
+
+type HistoryPageProps = React.FC & { getLayout: (page: HTMLElement) => JSX.Element };
+
+const HistoryPage: HistoryPageProps = () => {
+	return <History />;
```

Comment:
We should add a title to these pages:
<img width="239" alt="image" src="https://user-images.githubusercontent.com/10401554/184708872-70950d02-f259-472a-8b6b-afca80c333d2.png">